backend/src/middleware/slugifyMiddleware.spec.js

```diff
@@ -89,8 +89,8 @@ describe('slugify', () => {
   })
 
   describe('SignupVerification', () => {
-    const mutation = `mutation($password: String!, $email: String!, $name: String!, $slug: String, $nonce: String!, $termsAndConditionsAgreedVersion: String!) {
-      SignupVerification(email: $email, password: $password, name: $name, slug: $slug, nonce: $nonce, termsAndConditionsAgreedVersion: $termsAndConditionsAgreedVersion) { 
+    const mutation = `mutation($password: String!, $email: String!, $name: String!, $slug: String, $nonce: String!, $termsAndConditionsAgreedVersion: String!, termsAndConditionsAgreedAt: String) {
```

Comment:
No, the `termsAndConditionsAgreedAt` should *not* be supplied by the user. We don't care what the user tells us, what date it is. We **know** the date *on the server*. So, in the `SignupVerification` resolver, we set the `termsAndConditionsAgreedAt` manually.